test/passing/dune.inc

```diff
@@ -2015,12 +2015,14 @@
 
 (rule
  (deps .ocamlformat dir1/dir2/.ocamlformat dir1/dir2/print_config.ml)
+ (enabled_if (<> %{os_type} Win32))
  (action
    (with-outputs-to print_config.ml.output
      (run %{bin:ocamlformat} dir1/dir2/print_config.ml --print-config --config=max-iters=2 %{dep:print_config.ml}))))
```

Comment:
Would this one work if it was passed like this ? `%{dep:dir1/dir2/print_config.ml}`

Reply:
I modified this line, let's wait to see if it works on windows indeed